docs/zero-to-hero/01-basics/01-set-up-skeleton.md

```diff
@@ -0,0 +1,102 @@
+When you created your NEAR testnet account, a private key was created and placed into your browser's local storage. You may inspect this using developer tools and see it. 
+:::
+
+## Creating a new key on your computer
+
+We'll want to use a command-line interface (CLI) tool to deploy a contract, but at the moment the private key only exists in the browser. Next we'll _add a new key_ to the testnet account and have this stored locally on our computer as a JSON file. (Yes, you can have multiple keys on your NEAR account, which is quite powerful!)
+
+Let's install NEAR CLI. (Please ensure you [have NodeJS](https://nodejs.org/en/download/package-manager) > 12.)
+
+    npm install -g near-cli
+
+You may now run:
+
+    near
+
+to see various commands, which are covered [in detail here](https://docs.near.org/docs/tools/near-cli).
+
+We'll start by "logging in" with this command:
+
+    near login
+
+This will bring you to NEAR Wallet again where you can confirm the creation of a **full-access** key. We'll get to full-access and function-call access keys later, just know that for powerful actions like "deploy" we'll need a full-access key. Follow the instructions from the login command to create a key on your hard drive. This will be located in your operating system's home directory in a folder called `.near-credentials`.
+
+:::note How was a key added?
+When you typed `near login`, NEAR CLI generated a key pair: a private and public key. It kept the private key tucked away in a JSON file and sent the public key as a URL parameter to NEAR Wallet. The URL is long and contains other info instructing NEAR Wallet to "add a full access key" to the account. Our browser's local storage had a key (created when the account was made) that is able to do several things, including adding another key. It took the public key from the URL parameter, used it as an argument, and voilà: the testnet account has an additional key!
+:::
+
+You can see the keys associated with your account by running following command, replacing `friend.testnet` with your account name:
+
+    near keys friend.testnet
+
+## Setting up Rust
+
+You may have found the [online Rust Book](https://doc.rust-lang.org/stable/book), which is a great resource for getting started with Rust. However, there are a key items that are different when it comes to blockchain development. Namely, that smart contracts are [technically libraries and not binaries](https://learning-rust.github.io/docs/a4.cargo,crates_and_basic_project_structure.html#Crate), but for now just know that we won't be using some commands commonly found in the Rust Book.
+
+:::caution We won't be using
+    cargo run
+during smart contract development.
+:::
+
+Instead, we'll be iterating on our smart contract by building it and running tests.
+
+### Install Rust using `rustup`
+
+    curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+
+(Taken from the [Rust installation guide](https://www.rust-lang.org/tools/install))
```

Comment:
We should probably point people to install rustup instead from the [rustup.rs](rustup.rs) for it to point to windows installation too

Reply:
thanks, addressed